addons/mail_group/models/mail_group.py

```diff
@@ -0,0 +1,319 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import base64
+import logging
+from werkzeug import urls
+
+from ast import literal_eval
+
+from odoo import _, api, fields, models, modules
+from odoo.exceptions import ValidationError, UserError
+from odoo.tools import email_normalize, hmac
+
+_logger = logging.getLogger(__name__)
+
+
+class MailGroup(models.Model):
+    """Represent a mailing list.
+
+    The users can send emails to an alias and it will create <mail.group.message>. Those
```

Comment:
``Users send emails to an alias to create new discussions or reply to existing discussions. Moderation can be activated on groups. In that case email have to be validated or rejected.`` ?